app/models/enterprise.rb

```diff
@@ -97,8 +97,14 @@ class Enterprise < ApplicationRecord
   has_one_attached :promo_image
   has_one_attached :terms_and_conditions
 
-  validates :logo, content_type: %r{\Aimage/(png|jpeg|gif|jpg|svg\+xml|webp)\Z}
-  validates :promo_image, content_type: %r{\Aimage/(png|jpeg|gif|jpg|svg\+xml|webp)\Z}
+  validates :logo,
+             processable_image: true, # added processable_image: true
+             content_type: %r{\Aimage/(png|jpeg|gif|jpg|svg\+xml|webp)\Z}  
+
+  validates :promo_image, 
+              processable_image: true, # added processable_image: true
```

Comment:
The indentation doesn't look right here.

Reply:
indentation made confirmant to the style